exasol/toolbox/nox/_lint.py

```diff
@@ -177,3 +177,13 @@ def import_lint(session: Session) -> None:
             "Please make sure you have a configuration file for the importlinter"
         )
     _import_lint(session=session, path=path)
+
+
+@nox.session(name="lint:build-packages", python=False)
+def dist_check(session: Session) -> None:
+    """Checks whether your distribution’s long description will render correctly on PyPI
+
+    This has more robust checks for rst documentation than markdown.
```

Comment:
It's not in the official documentation, but I did several tests that showed it's not detecting errors with markdown. Feel free to try out yourself. In my commit, I switched the exasol-toolbox README.md to README.rst.

Please convert the `project-template/{{cookiecutter.repo_name}}/README.md` to be `project-template/{{cookiecutter.repo_name}}/README.rst`.